maybe the x4 needed for boobies to get the same value as Active(file) is due to #define MAX_NR_ZONES 4
diff --git a/fs/proc/meminfo.c b/fs/proc/meminfo.c
index 568d90e17c17..03ec679d3fc6 100644
--- a/fs/proc/meminfo.c
+++ b/fs/proc/meminfo.c
@@ -70,6 +70,8 @@ static int meminfo_proc_show(struct seq_file *m, void *v)
 	show_val_kb(m, "Active(anon):   ", pages[LRU_ACTIVE_ANON]);
 	show_val_kb(m, "Inactive(anon): ", pages[LRU_INACTIVE_ANON]);
 	show_val_kb(m, "Active(file):   ", pages[LRU_ACTIVE_FILE]);
+	show_val_kb(m, "Active(file)2:   ", global_node_page_state(NR_ACTIVE_FILE));
+	seq_printf(m, "Boobies: %5u\t%5lu \n", NR_ACTIVE_FILE/*it's 3, ie. nr[3]*/, global_node_page_state(NR_ACTIVE_FILE) * MAX_NR_ZONES);
 	show_val_kb(m, "Inactive(file): ", pages[LRU_INACTIVE_FILE]);
 	show_val_kb(m, "Unevictable:    ", pages[LRU_UNEVICTABLE]);
 	show_val_kb(m, "Mlocked:        ", global_zone_page_state(NR_MLOCK));
